src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/KeyWorkerResourceTest.kt

```diff
@@ -0,0 +1,133 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import com.microsoft.applicationinsights.TelemetryClient
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.isNull
+import org.mockito.kotlin.verify
+import org.springframework.boot.test.mock.mockito.SpyBean
+import org.springframework.http.MediaType
+
+class KeyWorkerResourceTest : ResourceTest() {
+
+  @SpyBean
+  protected lateinit var telemetryClient: TelemetryClient
```

Comment:
It would be good to get this SpyBean pushed up into a superclass so it can be shared among other tests  - this prevents Spring from restarting the app because it thinks the Context has changed.

Reply:
yeah - good idea - will do in separate PR